apps/site/assets/ts/stop/components/DepartureList.tsx

```diff
@@ -29,24 +29,35 @@ const DepartureList = ({
 
   let departures: DepartureInfo[] = [];
 
+  const tripForSelectedRoutePattern: Trip | null = schedules.length
+    ? schedules[0].trip
+    : null;
```

Comment:
nit-pick (non-blocking): This could be `schedules[0]?.trip` instead of the ternary statement